### DISCUSSION TITLE
v3: Frage: Was muss in der html drin stehen damit htmlTable genutzt werden kann?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#995] [Q&A]
```
[{"field":"Sender"},{"field":"Sendung"}]
```

In der html steht:
`<tr bgcolor= #0080ff><th>Sender</th><th>Sendung</th><th>Beginn</th><th>Ende</th><th>Typ</th></tr>`

**Comment by Zefau** [comment]
Bitte für Fragen die Diskussionen nutzen.

**Comment by mcuiobroker** [comment]
Habe hinterher nochmal `<thead> </thead> <tbody> </tbody> `hinzugefügt zur html-Tabelle, das hatte aber auch keinen Erfolg.
